Diz.Core/serialization/xml_serializer/ProjectXMLSerializer.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.IO;
+﻿using System.Diagnostics.CodeAnalysis;
+using System.IO;
```

Comment:
this class will need a slightly fresh coat of paint to incorporate the newer IConfigurationContainer